### PR TITLE
async operation status for privatelink scopes - JEDI urgent

### PR DESCRIPTION
**Pull request by arolshan**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by bennage** [comment]
@arolshan I'm not an appropriate reviewer for this. Perhaps I was added by mistake?

**Comment by arolshan** [comment]
++@Azure Resource Manager RP API Review<mailto:armapireview@microsoft.com> can anyone please review this PR?
its urgent for JEDI

Thanks, Arik.

**Review by majastrz — APPROVED** [review]
Signed off from ARM side.

File: specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json [hunk 1]
```diff
@@ -284,6 +284,46 @@
...
+      "get": {
+        "x-ms-examples": {
+          "Get specific operation status": {
+            "$ref": "./examples/privateLinkScopeOperationStatuses.json"
+          }
+        },
+        "tags": [
+          "asyncOperations"
+        ],
+        "operationId": "GetAsyncOperationsStatus",
```
Comment (allenjzhang) [inline specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json]: Operation ID should be Noun_Verb. Suggest **PrivateLinkScopeOperationStatus_Get**

File: specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json [hunk 2]
```diff
@@ -284,6 +284,46 @@
...
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Operation status.",
+            "schema": {
+              "$ref": "#/definitions/OperationStatus"
+            }
```
Comment (allenjzhang) [inline specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json]: Should have a default and refer to ErrorDetails under specification\common-type\v1\type.json. You can document the expected error code in the description section.

File: specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json [hunk 3]
```diff
@@ -764,6 +804,37 @@
...
+        "id": {
+          "type": "string",
+          "description": "The operation Id."
+        },
+        "name": {
+          "type": "string",
+          "description": "The operation name."
+        },
+        "startTime": {
+          "type": "string",
```
Comment (allenjzhang) [inline specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json]: Please add `        "format": "date-time",`  Same for endTime

File: specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json [hunk 4]
```diff
@@ -764,6 +804,37 @@
...
+          "description": "The start time of the operation."
```
Comment (allenjzhang) [inline specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json]: Please indicate time format in the description. Do you expect ISO8601 standard format?

File: specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json [hunk 5]
```diff
@@ -764,6 +804,37 @@
...
+        },
+        "endTime": {
+          "type": "string",
+          "description": "The end time of the operation."
+        },
+        "status": {
```
Comment (allenjzhang) [inline specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json]: nit: Is status supposed to be free forming text or more like an enum? If enum, please provide enum values.

**Review by allenjzhang — CHANGES_REQUESTED** [review]

File: specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json (hunk 1 above)
Comment (arolshan) [reply specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json]: thanks

File: specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json (hunk 4 above)
Comment (arolshan) [reply specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json]: thanks

File: specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json (hunk 2 above)
Comment (arolshan) [reply specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json]: didn't find it anywhere. thanks

File: specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json (hunk 5 above)
Comment (arolshan) [reply specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json]: free forming, thanks

File: specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json (hunk 2 above)
Comment (allenjzhang) [reply specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json]: See example in [hybridcompute](https://github.com/Azure/azure-rest-api-specs/blob/34ea001f2c8b49411863b571d3c66de8d80ca30b/specification/hybridcompute/resource-manager/Microsoft.HybridCompute/stable/2019-12-12/HybridCompute.json#L82)

Comment (akning-ms) [reply specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json]: Hi Arolshan, will you also change this? Looks you miss this one

**Comment by arolshan** [comment]
aligned to error response  from common as requested

**Review by allenjzhang — APPROVED** [review]

**Pull request merged**